Frontend/nutri-pal/src/app/diary/page.tsx

```diff
@@ -66,6 +87,18 @@ export default function diary() {
         }
     }
 
+    useEffect(() => {
+        console.log(filterDate)
+    }, [filterDate])
+
+    useEffect(() => {
+        console.log(filterYear)
+    }, [filterYear])
+
+    useEffect(() => {
+        console.log(filterMonth)
+    }, [filterMonth])
+
```

Comment:
When using the search across files method of vscode to search for console.logs, the word "console" was already on the search bar and I forgot to delete one letter in order for the search to update, but these console.logs were deleted in 2 commits ahead (8e97824ac701a518b0f8ed68c77dfaae182df126)